tests/e2e/common.go

```diff
@@ -97,6 +92,35 @@ func decodeJson(data []byte) (map[string]interface{}, error) {
 	return jsonData, err
 }
 
+func getJsonData(path string) (map[string]interface{}, error) {
+	// Return buffer data for json
+	jsonData, _ := ioutil.ReadFile(path)
```

Comment:
so the old getJsonData returned jsonData, err.  Is it safe to null the error here?